flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
I don't understand why this is still necessary. Isn't Java supposed to run everywhere?

Reply:
It is, but the build on aarch produces an unexpected error, and I don't know how to fix it at all